libs/ledger-live-common/src/hw/connectApp.ts

```diff
@@ -57,6 +66,16 @@ export type ConnectAppRequest = {
   requireLatestFirmware?: boolean;
   outdatedApp?: AppAndVersion;
   allowPartialDependencies: boolean;
+  passDeprecation?: boolean;
+};
+
+export type DeviceDeprecation = {
+  warningScreen: boolean;
+  warningClearSigning: boolean;
+  modelId: DeviceModelId;
+  date: string;
+  tokenExceptions: string[];
+  deprecatedFlowExceptions: string[];
 };
```

Comment:
Consider adding tsdoc comments to your types so everyone reading the code can understand exactly what it's for, how to use it etc.